### PR TITLE
[geolocator] Setup channel for geolocator service update

### PR DESCRIPTION
**Pull request by bbrto21**
Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

File: packages/geolocator/tizen/src/locaton_manager.h [hunk 1]
```diff
@@ -12,8 +12,14 @@
 #include "location.h"
 #include "tizen_result.h"
 
+// Defined in:
+
+// https://github.com/Baseflow/flutter-geolocator/blob/master/geolocator_platform_interface/lib/src/enums/location_service.dart
```
Comment (swift-kim) [inline packages/geolocator/tizen/src/locaton_manager.h:17]: Unexpected newline between the comments.

File: packages/geolocator/tizen/src/geolocator_tizen_plugin.cc [hunk 2]
```diff
@@ -44,6 +49,27 @@ class GeolocatorTizenPlugin : public flutter::Plugin {
   virtual ~GeolocatorTizenPlugin() {}
 
  private:
+  void SetupGeolocatorServiceUpdatesChannel(
+      flutter::BinaryMessenger *messenger) {
```
Comment (bwikbs) [inline packages/geolocator/tizen/src/geolocator_tizen_plugin.cc:53]: Suddenly curious...
``flutter::BinaryMessenger *messenger``
``flutter::BinaryMessenger* messenger``
Which one are we? doesn't it matter?

**Review by bwikbs — APPROVED** [review]
:+1:

File: packages/geolocator/tizen/src/geolocator_tizen_plugin.cc (hunk 2 above)
Comment (bbrto21) [reply packages/geolocator/tizen/src/geolocator_tizen_plugin.cc:53]: I think both are fine If you choose one consistently within a single file. and I always turn on auto formatting in vscode.
https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

Comment (swift-kim) [reply packages/geolocator/tizen/src/geolocator_tizen_plugin.cc:53]: The former is preferred by the Google style and the latter is used by the Chromium style. I don't know why. This repo follows the Google style so the former is correct.

Comment (bwikbs) [reply packages/geolocator/tizen/src/geolocator_tizen_plugin.cc:53]: That's a tragedy :cry:

File: packages/geolocator/tizen/src/locaton_manager.h (hunk 1 above)
Comment (bbrto21) [reply packages/geolocator/tizen/src/locaton_manager.h]: Thanks!

**Pull request merged**